examples/kubernetes/servicemesh/policy/external-lockdown.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumClusterwideNetworkPolicy
+metadata:
+  name: "external-lockdown"
+spec:
+  description: "Block all the traffic from external"
```

Grammar and wording suggestion:

```suggestion
  description: "Block all traffic originating from outside the cluster"
```